docs/concepts/storage/persistent-volumes.md

```diff
@@ -573,68 +573,53 @@ parameters:
 
 #### vSphere
 
-1. Create a persistent volume with a user specified disk format.
+   1. Create a persistent volume with a user specified disk format.
 
-```yaml
-kind: StorageClass
-apiVersion: storage.k8s.io/v1
-metadata:
-  name: fast
-provisioner: kubernetes.io/vsphere-volume
-parameters:
-  diskformat: zeroedthick
-```
+       ```yaml
+       kind: StorageClass
+       apiVersion: storage.k8s.io/v1
+       metadata:
+         name: fast
+       provisioner: kubernetes.io/vsphere-volume
+       parameters:
+         diskformat: zeroedthick
+       ```
 
--   `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
+       - `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
 
-2. Create a persistent volume with a disk format on a user specified datastore.
+    2. Create a persistent volume with a disk format on a user specified datastore.
+  
+        ```yaml
+        kind: StorageClass
+        apiVersion: storage.k8s.io/v1
+        metadata:
+          name: fast
+        provisioner: kubernetes.io/vsphere-volume
+        parameters:
+            diskformat: zeroedthick
+            datastore: VSANDatastore
+        ```
 
-```yaml
-kind: StorageClass
-apiVersion: storage.k8s.io/v1
-metadata:
-  name: fast
-provisioner: kubernetes.io/vsphere-volume
-parameters:
-    diskformat: zeroedthick
-    datastore: VSANDatastore
-```
+       -   `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
+       -   `datastore`: The user can also specify the datastore in the Storageclass. The volume will be created on the datastore specified in the storage class which in this case is `VSANDatastore`. This field is optional. If not specified as in previous YAML description, the volume will be created on the datastore specified in the vsphere config file used to initialize the vSphere Cloud Provider.
```

Comment:
StorageClass

If the datastore is not specified, then the volume will be....

vSphere

Reply:
Fixed!